### ISSUE TITLE
Unified time/date handling for ACCESS driver

### ISSUE THREAD

**Issue by aidanheerdegen** [OceansAus/access-om2#80]
As part of the manifest and file tracking (https://github.com/marshallward/payu/issues/90) I've been working on I find I would like a clearer delineation between inputs and restarts. Background: I've made a distinction between input files (which change rarely) and restarts (which we expect to change for every run).

As a solution to a previous issue (https://github.com/OceansAus/access-om2/issues/43) the access driver copies namelists from the output of a run to the restart directory. The cice driver opens the previous namelist to find out timing information:

https://github.com/marshallward/payu/blob/a2771de63085e3a3fa87a748004eae82f931b0cb/payu/models/cice.py#L161

I think it would be consistent (and cleaner) for the ice config to be copied to the ice working directory (which it is currently) and the timing information to be passed from an access level restart file. Could just be a f90 namelist file. As it is currently, the `cice_in.nml` is both a config file and a restart file.

Any comments @nicjhan?

**Comment by aidanheerdegen** [comment]
Had a discussion about this with @marshallward and @aekiss after the COSIMA meeting. The consensus was that this was an ok way to go for the moment, but a cleaner approach might be to extract the necessary fields from `cice_in.nml` and save them as a separate file in the `archive/restartXXX` directory, analogous to the `coupler.res` file in MOM5.

**Comment by rmholmes** [comment]
Just a quick comment to say that it would be nice not to require that the previous output folder exist in order to restart. I restart ACCESS-OM2 from restart's that I get from Kial quite often, where I don't have the corresponding output folder. I lost a few overnight runs because payu was looking for output***/ice/cice_in.nml when it didn't exist. It's fine now that I know what the problem is - I'm just creating an empty output***/ice/ with cice_in.nml from the ice configs.

**Comment by marshallward** [comment]
I agree with @rmholmes, drivers should only be looking in either restart* or the control directory for information, and anything needed for resubmission should be copied into restart*.

If access-om2 is doing this then we should submit a bug report.

**Comment by aidanheerdegen** [comment]
The old `access-om` driver fixed this, but I would encourage this to be thought through and not just copy files willy nilly. But I agree with the intention
